client/src/components/Contact.style.js

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+`;
+
+export const H1 = styled.h1`
+  font-weight: bold;
```

Comment:
you don't need
```font-weight: bold``` as you have already declared ```font-weight: 900``` 
below

also you still are missing a margin top on your h1! see here how the h1 goes into the header:
![Screen Shot 2019-06-27 at 16 37 41](https://user-images.githubusercontent.com/37771591/60279934-e6532c80-98f9-11e9-89ac-4a0cab1e4742.png)


I would suggest maybe changing ```margin-bottom``` to ```margin```